bin/setup_dev

```diff
@@ -62,7 +62,7 @@ $AG install \
     ca-certificates sudo net-tools tcpdump build-essential \
     isc-dhcp-client network-manager netcat gnupg2 strace \
     python3 python3-pkg-resources python3-setuptools python3-dev \
-    python3-pip openjdk-8-jdk python emacs-nox python3-venv libffi-dev jq pandoc
+    python3-pip openjdk-8-jdk python emacs-nox python3-venv libffi-dev python3-cairocffi jq pandoc
```

Comment:
maybe line-wrap at similar length to previous lines

Reply:
Sure, now done.